qt/widgets/instrumentview/src/Viewport.cpp

```diff
@@ -8,10 +8,8 @@
 #include "MantidGeometry/Rendering/OpenGL_Headers.h"
 #include "MantidKernel/V3D.h"
 
-#include <QtGlobal>
```

Comment:
So we had originally intended to remove this `TSV` serializer code and project saving as it was used for the old Qt MantidPlot project saving and workbench has a different mechanism to do this. It now uses a [separate encoder class](https://github.com/mantidproject/mantid/blob/cd501cac0766537854cf7ac395c26fbccb7cf4e9/qt/widgets/instrumentview/src/InstrumentWidgetEncoder.cpp) so the intention was that the project saving would be removed.

I think it's fine for Q1DWeighted to say it accepts a table of a given format, documented in it's documentation, and that we can produce that table here but can we decouple the old project saving/TSV serializer code at all?

Reply:
TSV serialiser is used to generate the content of the shape table workspace. We are not happy at all the way it's done currently, since it is not human-readable. We can get rid of this if we change the format of the table, however we were not sure if there is other client code that is relying on the existing format. If it is just us, then we are happy to redo the table formatting.

Reply:
Ah perfect! Now we have rid of MantidPlot this saving code is not being used and it was intended as an internal MantidPlot format for the projects so I think this is actually a good opportunity to improve the situation here and serialize this information in a fashion that is more easily consumed.

Reply:
Would you mind if we get this PR in first, then change the table format later as it would not alter the logic, just the I/O between the instrument viewer and the algorithm.

Reply:
Do we think we can get that done before the release? I'm mostly concerned that the official release has the "correct" format rather than then having to support both going forward on the algorithm.

Reply:
Is it worth putting a warning into Q1DWeighted that the format is not stable and this option is in active development?